spack-repo/packages/singularity-eos/package.py

```diff
@@ -96,8 +96,9 @@ class SingularityEos(CMakePackage, CudaPackage):
 
     # building/testing/docs
     depends_on("cmake@3.19:")
-    depends_on("catch2@3.0.1:", when="@main +tests")
     depends_on("catch2@2.13.7", when="@:1.8.0 +tests")
+    depends_on("catch2@3.0.1:", when="@1.9.0: +tests")
+    depends_on("catch2@3.0.1:", when="@main +tests")
```

Comment:
The `@main` line is not needed. `1.9.0:` is "1.9.0 and anything newer, which `main`, `master` and `develop` always are.

Reply:
Done